tests/test_rest.py

```diff
@@ -0,0 +1,613 @@
+import unittest
+import json
+
+import mock
+import requests
+
+from pricecypher.rest import RestClient, RestClientOptions
+from pricecypher.exceptions import PriceCypherError, RateLimitError
+
+
+class TestRest(unittest.TestCase):
+    TIMEOUT_DEF = 300.0
+
+    @classmethod
+    def setUpClass(cls):
+        cls.low_timeout_options = RestClientOptions(0.00001)
+        cls.tuple_timeout_options = RestClientOptions((10, 2))
+        cls.static_response_headers = {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json',
+        }
+
+    def test_get_can_timeout(self):
```

Comment:
These test_X_can_timeout tests are not working for me. For this specific one, it does not work for `google.com`, nor for `speedtest.net` or `netflix.com`, but does work for `nytimes.com` and `reddit.com` (both low ping too) and `amazon.com` (>100ms ping).

Reply:
Are you inside a google datacenter 😮? I cannot imagine you get a response within 0.01 milliseconds, so something else must be going wrong in the test. However, from your comment it's not clear to me what exactly. Maybe let's resolve this when we setup a CI pipeline for this.

Reply:
Well, I was pretty much on site of an Internet Exchange point 🙃, but performing the same tests at home gives the same results.